.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+
+tag-pattern: &tag-pattern
+  only: /^v\d+\.\d+\.\d+$/
+
+jobs:
+  test:
+    working_directory: ~/rn-salesforce-chat
+    docker:
+      - image: circleci/node:11.10.1
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: yarn-v1-{{ checksum "yarn.lock" }}-{{ arch }}
+
+      - restore_cache:
+          key: node-v1-{{ checksum "package.json" }}-{{ arch }}
+
+      - run: yarn install
+
+      - save_cache:
+          key: yarn-v1-{{ checksum "yarn.lock" }}-{{ arch }}
+          paths:
+            - ~/.cache/yarn
+
+      - save_cache:
+          key: node-v1-{{ checksum "package.json" }}-{{ arch }}
+          paths:
+            - node_modules
+
+      - run:
+          name: jest tests
+          command: yarn test --maxWorkers=2
+
+  publish:
+    working_directory: ~/rn-salesforce-chat
+    docker:
+      - image: circleci/node:11.10.1
+
+    steps:
+      - checkout
+
+      - run:
+          name: Authenticate npm with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
```

Comment:
@barbosa @pedrohperalta Do you think it is possible for us to create/use a Loadsmart npm account, so we can automate the release of this library?

Reply:
@lenoirzamboni we do have one: https://www.npmjs.com/~loadsmart-frontend

You can check how this is being done on our icons library.
The token you get it "for free" if you say in your CircleCI config file that you want to use the `org-global` [context](https://app.circleci.com/settings/organization/github/loadsmart/contexts/3699e01f-845b-4eaa-836c-f6ac9b8a5749?return-to=https%3A%2F%2Fcircleci.com%2Fdashboard).

Reply:
@barbosa Turns out we are already using org-global context.